app/classes/document_search_results.rb

```diff
@@ -42,8 +44,6 @@ def extract_hits(hits)
 
   def deserialized(hit)
     Serde.deserialize_hash(ActiveSupport::HashWithIndifferentAccess.new(hit['_source']),
-                           hit['_source']['language'],
-                           %i[title description content])
```

Comment:
I have slightly refactored the Serde methods so that they no longer take the "language field keys" (`%i[title description content]`) as an argument.